tests/unittests/atom/molmass_test.py

```diff
@@ -0,0 +1,16 @@
+"""test for molmass calculation"""
+
+import pytest
+from exojax.spec.molinfo import molmass
+
+
+def test_molmass():
+    molmass('air')
```

Comment:
(if I can suggest, you should add `assert` statements there
ex: 
```python
assert molmass('air')== 28.97
```

Reply:
They'll raise an AssertionError if it doesn't work. 

Advanced note : sometimes (not here) you want to test that an error is raised, i.e., the normal behavior is to expect an error (for instance, you're trying to catch a wrong user input, etc.) 
You can use 
```python 
with pytest.raises(ValueError):
    this_code()   #  test will pass IF this_code() raises a ValueError
```

Reply:
Thank you very much for your suggestion and helpful tip! I will implement it.